### PR TITLE
CORDA-3240: optional node.conf property not recognized when overridden

### PR DESCRIPTION
**Pull request by bpaunescu**
If the networkParameterAcceptanceSettings (has default values) is overridden in node.conf, the node fails to start due to not recognizing it (not added in the schema validation class)

### CODE REVIEW

File: settings.gradle [hunk 1]
```diff
@@ -16,7 +16,7 @@ include 'finance:workflows'
 include 'isolated'
 include 'core'
 include 'core-tests'
-include 'docs'
+//include 'docs'
```
Comment (oknowles) [inline settings.gradle]: Was this change meant to go into this PR? Seems unrelated

Comment (bpaunescu) [reply settings.gradle]: Sorry, slipped. I usually comment that out when building locally.

**Review by oknowles — APPROVED** [review]
LGTM
